scripts/bozzcoin.coffee

```diff
@@ -79,9 +95,12 @@ module.exports = (robot) ->
         if repsDone > 13000
           res.send "Are you sure you can play longer than :pohneo:?"
           return
+    if repsDone > 0 && updateBozziplier(username)
+      res.send ":bozz: Bozziplier reset like a bozz"
+    bozziplier = robot.brain.get("bozziplier")
     newBozzcoinBalance = robot.brain.get("bozzcoinBalance") + convertToBozzcoin(repsDone, exerciseType)
     robot.brain.set("bozzcoinBalance", newBozzcoinBalance)
-    res.send "#{res.match[1]} #{exerciseType} done, *#{newBozzcoinBalance}* :bozzcoin: available!"
+    res.send "#{res.match[1]} #{exerciseType} done with bozzplier of #{bozziplier}, *#{newBozzcoinBalance}* :bozzcoin: available!"
```

Comment:
Should be spelt `bozziplier` with the middle `i`.

Also, can add the amount of bozzcoin earned? Something like:
```
10 pullups done, earned 100 :bozzcoin: with bozziplier of 1.
4125 :bozzcoin: available.
```

Reply:
Fixed and modified.